datasafe-simple-adapter/datasafe-simple-adapter-impl/src/main/java/de/adorsys/datasafe/simple/adapter/impl/pathencryption/NoPathEncryptionModule.java

```diff
@@ -14,7 +14,7 @@
  * This module is responsible for providing pathencryption of document.
  */
 @Module
-public abstract class SwitchablePathEncryptionModule {
+public abstract class NoPathEncryptionModule {
```

Comment:
SwitchablePathEncryptionModule was constructed by swagger. I have seen no way to pass it a spring configuration. Thats why I removed the SwitchablePathModule completly. Now DefaultDatasafeServuces in SimpleDatasafeServiceImpl become constructed with the default PathEncryptionModule - UNLESS spring tells it to us NoPathEncryptionModule. What is wrong with this approach?

Reply:
The point is either there is no need in this module at all because if the module is active it disables path encryption (and it is already `disablable` by overridesRegistry) or something is wrongly configured if the module is needed